guides/release/accessibility/learning-resources.md

```diff
@@ -1,31 +1,32 @@
-These accessibility learning resources will provide additional support to the developer looking to improve the quality of the code they write. 
+Ces ressources d'apprentissage sur l'accessibilité ont pour but d'apporter une aide additionnelle aux développeurs.euses pour améliorer la qualité de leur code.
 
-- [MDN Web Docs: Accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
-- [Using ARIA:](https://www.w3.org/TR/using-aria/) a practical guide for developers on how to add accessibility information to HTML elements
-- [Web Content Accessibility Guidelines(WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
-- [Accessible Rich Internet Applications (ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/) 
-- [How to meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
+*Note: Les liens ci-dessous redirigent vers des articles écrits en Anglais
 
+- [Documentation Web MDN: Accessibilité](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
+- [Utiliser ARIA:](https://www.w3.org/TR/using-aria/) un guide pratique pour développeurs.euses: Comment ajouter des informations d'accessibilité sur des éléments HTML
+- [Web Content Accessibility Guidelines(WCAG): Directives du contenu web sur l'accessibilité 2.1](https://www.w3.org/TR/WCAG21/)
```

Comment:
Do we want to keep some terms in English? Here I kept Web Content A11y Guidelines as it's the abbreviation of WCAG

Reply:
Answered in the suggestions 👍

Reply:
Thank you for the thorough review!